rfc-158-port-content-store-to-postgresql.md

```diff
@@ -0,0 +1,49 @@
+# Port Content-Store to PostgreSQL on RDS
+
+## Summary
+
+GOV.UK Publishing should port content-store from its current legacy self-hosted MongoDB, to PostgreSQL running on AWS' RDS.
```

Comment:
Nitpick: I believe we usually refer to apps as a proper noun ('Content Store'), only using kebab-case to refer to the repo ('content-store').

```suggestion
GOV.UK Publishing should port [Content Store](https://github.com/alphagov/content-store) from its current legacy self-hosted MongoDB, to PostgreSQL running on AWS' RDS.
```

Reply:
Nitpick: we should also apply the same tweak consistently throughout the RFC @aldavidson 👍

Reply:
Done!

Reply:
Thanks!